.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -2,3 +2,6 @@ contact_links:
   - name: Questions 🤔
     url: https://github.com/iTwin/iTwinUI/discussions/categories/q-a
     about: Ask questions and get help from contributors through GitHub Discussions.
+  - name: Request an example 👀
+    url: https://github.com/iTwin/iTwinUI/discussions/categories/examples
```

Comment:
This link opens the "Examples" page showing existing discussions, but we also have the ability to pre-fill the form using a link like this: https://github.com/iTwin/iTwinUI/discussions/new?category=examples&title=blah&body=pls%20show%20me%20an%20example%20of

Not sure how useful that would be, so I went with the simpler link.